models/staging/stg_page_definitions__all.sql

```diff
@@ -0,0 +1,13 @@
+select
+  CAST(id AS int64) as id,
```

Comment:
Keyword casing consistency is another thing linters might help with.

Personally, I prefer lowercase because it's not 1970 anymore.